packages/frontend/src/components/TestSubstep/index.tsx

```diff
@@ -149,6 +160,7 @@ function TestSubstep(props: TestSubstepProps): JSX.Element {
         expanded={expanded}
         onClick={onToggle}
         title={substep.name}
+        rightEl={<TestSubstepTitleTooltip />}
```

Comment:
```suggestion
        rightEl={<TitleTooltip />}
```
the component name sounds damn long but just a suggestion

Reply:
it is but it's only temporary. will remove when sst is stable